src/CommonITILObject.php

```diff
@@ -9170,11 +9172,21 @@ public static function getDataToDisplayOnKanban($ID, $criteria = [])
             }
         }
 
+        $category_names = [];
         foreach ($common_itil_iterator as $data) {
+            if (!isset($category_names[$data['itilcategories_id']])) {
+                $category_names[$data['itilcategories_id']] = DropdownTranslation::getTranslatedValue(
+                    $data['itilcategories_id'],
+                    ITILCategory::class,
+                    'name',
+                    $_SESSION['glpilanguage'],
+                    ITILCategory::getFriendlyNameById($data['itilcategories_id'])
+                );
```

Comment:
This will prevent to do an extra query in `ITILCategory::getFriendlyNameById()` when there is a translation for the category name.

```suggestion
                $category_name = DropdownTranslation::getTranslatedValue(
                    $data['itilcategories_id'],
                    ITILCategory::class
                );
                if ($category_name === '') {
                    $category_name = ITILCategory::getFriendlyNameById($data['itilcategories_id'])
                }

                $category_names[$data['itilcategories_id']] = $category_name;
```